lib/views/invoice_slack_view.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Aprb.Views.InvoiceSlackView do
+  import Aprb.ViewHelper
+
+  def render(event) do
+    partner_data = fetch_partner_data(event["properties"]["partner_id"])
+    %{
+      text: ":money_with_wings: Invoice #{event["object"]["display"]} #{event["verb"]}",
+      attachments: "[{
```

Comment:
What about refactoring the JSON into an object that's then Poison-encoded? Something like:

```ex
data = [ %{ text: "...", attachements: "..." ] 
data |> Poison.encode
```